dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -133,10 +146,298 @@ xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[VIP] ad
+
+[[using-ingress-defining-the-ingress-IP-range]]
+==== Defining the Ingress IP Range
+
+The ability for cluster administrators to automatically allocate External IP
+addresses using Ingress is enabled by default within {product-title} and
+configured to use the `172.46.0.0/16` range. An alternate range can be specified
+by configuring the `ingressIPNetworkCIDR` parameter in the
+*_/etc/origin/master-config.yaml_* file:
+
+----
+networkConfig:
+  ingressIPNetworkCIDR: 10.9.54.0/25
+----
+
+Restart the {product-title}  master service to apply the changes.
+
+[[using-ingress-deploy-a-sample-application]]
+==== Deploy a Sample Application
+
+To expose a PostgreSQL as a service for external consumption, the application
+must be first deployed into the cluster. Create a new project or use and
+existing project and instantiate one of the PostgreSQL templates.
+
+[CAUTION]
+====
+The *postgresql-ephemeral* template does not make use of persistent storage. Once
+the application is scaled down or destroyed, any existing data will be lost. To
+use persistent storage, specify the *postgresql-persistent* template instead.
+====
+
+After instantiating the template, a ClusterIP-based service and
+`DeploymentConfig` is created and a new pod containing PostgreSQL will be
+started.
+
+[[configuring-an-IP-address-for-a-service]]
+==== Configuring an IP Address for a Service
+
+To allow the cluster to automatically assign an IP address for a service, create
+a service definition similar to the following that will create a new Ingress
+service:
+
+----
+$ oc create -f - <<INGRESS
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+spec:
+  ports:
+  - name: postgresql
+    port: 5432
+  type: LoadBalancer <1>
+  selector:
+    name: postgresql
+INGRESS
+----
+<1> The `LoadBalancer` type of service will make the request for an external service
+on behalf of the application user.
+
+Alternatively, the `oc expose` command can be used to create the service:
+
+----
+$ oc expose dc postgresql --type=LoadBalancer --name=postgresql-ingress
+----
+
+Once the service is created, the external IP address is automatically allocated
+by the cluster and can be confirmed by running:
+
+----
+$ oc get svc postgresql-ingress
+----
+
+.Example oc get Output
+----
+NAME         CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+postgresql-ingress    172.30.74.106   10.9.54.100,10.9.54.100    5432/TCP    30s
+----
+
+Specifying the type `LoadBalancer` also configures the service with a
+`nodePort` value, allowing direct access to the service on each of the nodes the
+application is deployed on. To discover the node port automatically assigned,
+run:
+
+----
+$ oc export svc postgresql-ingress
+----
+
+.Example oc export Output
+----
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    app: postgresql-persistent
+    template: postgresql-persistent-template
+  name: postgresql-ingress
+spec:
+  deprecatedPublicIPs:
+  - 10.9.54.100
```

Comment:
@rjhowe do you know what depricated public IP represents? I could not find any other reference to it. Sourced from the example content you pointed me to.